website/source/layouts/docs.erb

```diff
@@ -16,6 +16,10 @@
         </ul>
       </li>
 
+      <li<%= sidebar_current("docs-glossary") %>>
```

Comment:
This section feels like it's in a weird spot. I think it would be better next to architecture, or perhaps both it and architecture should go in the section with the FAQ and common error messages 🤔

Reply:
I'm ok with putting the new glossary page just above internals. I would like it to be at the top of the page, before the fold. If we move it to the FAQ section then its likely below the fold and may be less discoverable.